src/Goerli-DssSpell.t.sol

```diff
@@ -163,31 +198,40 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         assertTrue(lerp.done());
     }
 
-    function testNewChainlogValues() private { // make public to use
+    function testNewChainlogValues() public { // make private to disable
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // Insert new chainlog values tests here
-        // checkChainlogKey("MCD_JOIN_TELEPORT_FW_A");
-        // checkChainlogVersion("1.14.0");
+        // RWA007
+        checkChainlogKey("RWA007_A_JAR");
+        checkChainlogKey("RWA007");
+        checkChainlogKey("MCD_JOIN_RWA007_A");
+        checkChainlogKey("RWA007_A_URN");
+        checkChainlogKey("RWA007_A_INPUT_CONDUIT_URN");
+        checkChainlogKey("RWA007_A_INPUT_CONDUIT_JAR");
+        checkChainlogKey("RWA007_A_OUTPUT_CONDUIT");
+        checkChainlogKey("PIP_RWA007");
+
+        checkChainlogVersion("1.14.2");
     }
 
-    function testNewIlkRegistryValues() private { // make public to use
+    function testNewIlkRegistryValues() public { // make private to disable
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // Insert new ilk registry values tests here
-        // assertEq(reg.pos("TELEPORT-FW-A"), 52);
-        // assertEq(reg.join("TELEPORT-FW-A"), addr.addr("MCD_JOIN_TELEPORT_FW_A"));
-        // assertEq(reg.gem("TELEPORT-FW-A"), address(0));
-        // assertEq(reg.dec("TELEPORT-FW-A"), 0);
-        // assertEq(reg.class("TELEPORT-FW-A"), 4);
-        // assertEq(reg.pip("TELEPORT-FW-A"), address(0));
-        // assertEq(reg.xlip("TELEPORT-FW-A"), address(0));
-        // assertEq(reg.name("TELEPORT-FW-A"), "");
-        // assertEq(reg.symbol("TELEPORT-FW-A"), "");
+        // RWA007
+        (, address pipRwa007,,) = oracle.ilks("RWA007-A");
+
+        assertEq(reg.pos("RWA007-A"),    53);
+        assertEq(reg.join("RWA007-A"),   addr.addr("MCD_JOIN_RWA007_A"));
+        assertEq(reg.gem("RWA007-A"),    addr.addr("RWA007"));
+        assertEq(reg.dec("RWA007-A"),    GemAbstract(addr.addr("RWA007")).decimals());
+        assertEq(reg.class("RWA007-A"),  3);
+        assertEq(reg.pip("RWA007-A"),    pipRwa007);
```

Comment:
this should be checked against `addr.addr("PIP_RWA007")`

Reply:
yeah, if the pip have been precomputed and added to `addresses_goerli.sol` the check above could be extended to ensure that both created and stored pip in `addresses_goerli.sol` match, otherwise if placeholders were used in previous spells in `addresses_goerli.sol`, they should be updated to match the chainlog or the precomputed pip.

Reply:
actually checking the pip in the registry against the `address_goerli.sol` should be enough

Reply:
`test_chainlog_values` should ensure that previous pips were precomputed and match the chainlog, so I would suggest to check directly against `addresses_goerli.sol`